src/stores/wallet.ts

```
@@ -0,0 +1,202 @@
+  // removeWallet(): void
+  // setWallet(wallet: Wallet): void
+  // setBalance(to: string): void
+  // setLedgerBluetooth(ledgerBLE: boolean): void
+  // setWeb3Instance(web3: unknown): void
+  // setOwnedDomains(ensDomains: string[]): void
+  // setTokens(tokens: unknown[]): void
+  // setLoadingWalletInfo(isLoading: boolean): void
+  // setOfflineApp(isOfflineApp: boolean): void
+  // setLedgerApp(ledgerApp: Record<PropertyKey, unknown>): void
+  // setSwapRates(swapRates: unknown[]): void
+}
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)